collectors/charts.d.plugin/charts.d.plugin.in

```diff
@@ -410,6 +410,7 @@ declare -A obsolete_charts=(
 	['postfix']="python.d.plugin module"
 	['squid']="python.d.plugin module"
 	['tomcat']="python.d.plugin module"
+	['apcupsd']="temporary disabled because of the bug (#12413)"
```

Comment:
This leaves no way to enable. Why no put this in `charts_enable_keyword` instead?

Reply:
@ilyam8 The bug has been in the collector for some time now, we just happened to have it show up unexpectedly here because of the change in packages in the Docker images pulling in the required tooling for the collector to even try to run. People _have_ been using the collector without seeing this issue.

Given all of this, it’s safe to assume that the bug _only_ happens when the tooling is installed _and_ there is no valid configuration for said tooling. Which means that we should not be globally disabling it for everyone, just flipping it to default to not running even if it sees the tooling (because again, people _are_ using it).